CHANGELOG.md

```diff
@@ -281,7 +281,7 @@
 - PR #5914 Link CUDA against libcudf_kafka
 - PR #5895 Do not break kafka client consumption loop on local client timeout
 - PR #5915 Fix reference count on Java DeviceMemoryBuffer after contiguousSplit
-
+- PR #5942 Fix JIT cache multiprocess test failure
```

Comment:
In the rest of this file, there are 2 blank lines between the Bug section and the previous release header.